analytics/terraform/spark-k8s-operator/jupyterhub.tf

```diff
@@ -0,0 +1,49 @@
+#-----------------------------------------------------------------------------------------
+# JupyterHub Single User IRSA Configuration
+#-----------------------------------------------------------------------------------------
+resource "kubernetes_namespace" "jupyterhub" {
```

Comment:
These resources requires adding a condition `enable_jupyterhub` so that we created these resources only when it's true.